guides/MoodleIntegrationGuide.md

```diff
@@ -125,28 +173,52 @@ The next step in the integration process is to download the openEQUELLA Moodle M
 This section describes the required configuration changes to Moodle to support the integration. openEQUELLA provides a module that allows Moodle users to easily contribute, search and select openEQUELLA items, and link them to a course. 
 
 ### Download and install the openEQUELLA Moodle Modules
+
 The Moodle Modules are available from the Downloads page in the openEQUELLA Github repository
+The Moodle Modules are available from each of their Github repositories:
+
+* Moodle module - <https://github.com/openequella/moodle-mod_openEQUELLA>
+* Moodle search block - <https://github.com/openequella/moodle-block_openEQUELLA_search>
+* Moodle tasks block - <https://github.com/openequella/moodle-block_openEQUELLA_tasks>
+* Moodle links block - <https://github.com/openequella/moodle-block_openEQUELLA_links>
+* Supporting diagnostic tool - <https://github.com/openequella/moodle-mod_openEQUELLA-tools>
 
-To download the Moodle Modules
-1.  Navigate to the [Downloads Page](../IntegrationsDownloads.md) in the openEQUELLA Github Repository
+To download the main Moodle Module, simple download the [latest
+release](https://github.com/openequella/moodle-mod_openEQUELLA/releases/latest) from the [releases
+page](https://github.com/openequella/moodle-mod_openEQUELLA/releases). For the blocks and the
+diagnostic tool, you will need to go to use the green Github 'Clone or download' button.
```

Comment:
From the start of the section to here was the main trigger for me doing this - to ensure people could find their way to the new downloads etc.